src/host/main.cpp

```diff
@@ -297,6 +297,16 @@ int main(int argc, char** argv)
   app.add_option(
     "--domain", domain, "DNS to use for TLS certificate validation");
 
+  std::string subject_name("CN=CCF Node");
+  app.add_option("--sn", subject_name, "Subject Name in node certificate, eg. CN=CCF Node")
+    ->capture_default_str();
+
+  std::vector<std::string> subject_alternative_names;
+  app.add_option(
```

Comment:
Should we use a special parser here, similarly to the address options, to check that the values are in the expected format?

Reply:
I hadn't thought of that, but absolutely, yes!